lnrpc/lightning.proto

```diff
@@ -3841,7 +3841,30 @@ message Invoice {
     Signals that the invoice should include blinded paths to hide the true
     identity of the recipient.
     */
-    bool blind = 29;
+    BlindedPathConfig blinded_path_config = 29;
```

Comment:
Proto comment should be updated so it reflects in the API docs.

Reply:
Hmm, I see why you went in this direction, but it's a bit of a deviation from the other `bool` flags we have such as `private`: to just activate the feature, an _empty_ proto can/should be set vs a `bool`.

Reply:
yeah the reason I went this way is to keep 1 source of truth at the protobuf level. Ie instead of allowing a user to set a `blind` bool to false but then also set config options to non-null, just dont allow that and have it be all-in-one

Reply:
Cool, I think let's wait for some app/wallet level feedback, then we can tune things based on that.